tests/container_test.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env pytest -vs
+"""Tests for example container."""
+# TODO: Make container tests functional
+# See https://github.com/cisagov/pshtt_reporter/issues/45
+
+# Standard Python Libraries
+import os
+
+# import time
+
+# import pytest
+
+ENV_VAR = "ECHO_MESSAGE"
+ENV_VAR_VAL = "Hello World from docker-compose!"
+READY_MESSAGE = "This is a debug message"
+SECRET_QUOTE = (
+    "There are no secrets better kept than the secrets everybody guesses."  # nosec
+)
```

Comment:
Should we just remove these since they don't seem like they'd be used for testing this container configuration anyway?

```suggestion
```